### PR TITLE
Changes and rectification

### PR DESCRIPTION
**Pull request by Tuasegun**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,8 +57,15 @@
     "yarn-run-all": "^3.1.1"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.36",
```
Comment (ickynavigator) [inline package.json:60]: put the packages in the package.json you are actually using them

**Review by ickynavigator — CHANGES_REQUESTED** [review]

**Comment by ickynavigator** [comment]
resolve conflicts

File: package.json (hunk 1 above)
Comment (Tuasegun) [reply package.json:60]: I'm actually not using them, I use react-icons. I needed my work to run so I just installed them when I noticed it was missing

Comment (ickynavigator) [reply package.json:60]: check which spa uses it so you can move them into that

**Pull request merged**